live_data_subscriber/websocket/test_scripts/ws_server.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import logging
+import websockets
+from websockets import WebSocketServerProtocol
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Server:
```

Comment:
Please find a less generic way for this class. 
I would say at least `WSServer`.

Reply:
OK, thanks for the offline discussion. This is minor then (as this is only for testing)